### DISCUSSION TITLE
Edited

### DISCUSSION THREAD

**Discussion by Phoenix-H22** [SafeExamBrowser/seb-win-refactoring#244] [Q&A]
We are an educational Organization and we use Safe Exam Browser to examine our students and we need to make an edited version for our school so we ask if we can pay for that and make a specific version for us are you accept that or not?

**Comment by dbuechel** [comment]
Unfortunately, we cannot make a specific edited version of SEB for you due to the fact that we are not a company and we do not sell anything. Please refer to our FAQ on this topics: https://safeexambrowser.org/alliance/faq.html. However, you may hire developers or have contractors work on a special edition of SEB for you. The MPL license  (https://www.mozilla.org/en-US/MPL/1.1/) under which SEB is released allows this, given the conditions listed in the license.
